Syllabi/misc.md

```diff
@@ -0,0 +1,56 @@
+# Miscellaneous thoughts on assessment
+
+## What should we teach?
+
+* We've mostly spoken about software tools (i.e., Git, GitHub, Docker,
+  Python, R, Make, unit tests, etc.) for reproducibility.  Yet the mechanical
+  reproduction of results is not an end, but a means to provide a secure
+  foundation for reliable research.  To what extent
+  should we be teaching/discussing other aspects of computational
+  reproducibility (e.g., statistical issues) and tool-agnostic principles
+  that can be introduced in many different contexts.
+* Statistical training makes sense for many scientists, but may
+  not be as widely appropriate as the tools for computational reproducibility.
+  For instance, what role would statistical concerns play in a traditional
+  numerical analysis course. The specifics of statistical reproducbility
+  are probably best determined at a domain specific level.
```

Comment:
There might still be some statistical issues that cross domains. For example, the relationship between effect size and probability of reproducing that result. Once the philosophy is understood, this can also be described.